packages/vm/lib/evm/evm.ts

```diff
@@ -255,10 +255,19 @@ export default class EVM {
       totalGas = totalGas.add(returnFee)
     }
 
-    // if not enough gas
+    // Check for SpuriousDragon EIP-170 code size limit
+    const codeSizeLimit = 24576 // 0x600
```

Comment:
I suggest we add this value in common

Reply:
Ah, just discovered, this [is actually integrated](https://github.com/ethereumjs/ethereumjs-vm/blob/master/packages/common/src/hardforks/spuriousDragon.json#L16) in Common. Lol. Will update.

Reply:
Have done.